README.md

```diff
@@ -44,27 +44,34 @@ npm config set externalplugins ../relative/path/to/plugins/directory
 
 ## Adding npm Package Dependencies
 
-If you want to add a new npm package to 'node_modules':
+If you want to add a new npm package to 'node_modules' you will need to `--save-exact`:
 
 1. Install the new package
-  * Install and save dependencies in `package.json`
+  * The command below will install and save dependencies in `package.json`
 
     ```
-    npm install [your package] --save
+    npm install [your package] --save --save-exact
     ```
 
-  * Then, add the package to devDependencies
+  * Then, (if needed) add the package to devDependencies
 
     ```
-    npm install [your package] --save-dev
+    npm install [your package] --save-dev --save-exact
     ```
 
 2. Create a synced npm-shrinkwrap.json with devDependencies included by running
 
     ```
+    npm shrinkwrap
```

Comment:
Nope, you always want to run it with the `--dev` flag, that's why we have the `shrinkwrap` script in package.json and not just have you remember it every time. So this needs to be `npm run shrinkwrap`.